### ISSUE TITLE
Metrics giving decimal format result

### ISSUE THREAD

**Issue by kungfupanda792** [thought-machine/aws-service-quotas-exporter#16]
So this metrics aws_available_ips_per_subnet_used_total is gving results in decimal format

pod="aws-quota-checker"      0.005859375
                                                   0.1875

  this is decimal values we are getting when we are using the above metric

**Comment by idgenchev** [comment]
Hello @kungfupanda792 this is fixed in [v1.2](https://github.com/thought-machine/aws-service-quotas-exporter/releases/tag/v1.2) as part of #14. Which version of the aws-service-quotas-exporter are you using?

**Comment by idgenchev** [comment]
I can confirm that's been fixed in [v1.2](https://github.com/thought-machine/aws-service-quotas-exporter/releases/tag/v1.2). So simply upgrading to that version will resolve your issue. Thanks for submitting a GitHub issue!

Here is an example response for both:
```
aws_available_ips_per_subnet_used_total{region="eu-west-1",resource="subnet-03f6e727438a1b1c9"} 7959
aws_available_ips_per_subnet_limit_total{region="eu-west-1",resource="subnet-03f6e727438a1b1c9"} 8192
```
